### PR TITLE
frontend: fix nullable node status

### PR DESCRIPTION
**Pull request by farodin91**

### CODE REVIEW

File: frontend/src/components/cluster/Charts.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ export function MemoryCircularChart(props: ResourceCircularChartProps) {
   }
 
   function memoryAvailableGetter(item: Node | Pod) {
-    return parseRam(item.status!.capacity.memory) / TO_GB;
+    return parseRam(item.status!.capacity?.memory) / TO_GB;
```
Comment (sniok) [inline frontend/src/components/cluster/Charts.tsx:22]: might as well remove the `!` here `item.status?.capacity?.memory`

**Comment by sniok** [comment]
nice, thanks for adding this @farodin91. Was this change prompted by some issue you've experienced?

**Comment by farodin91** [comment]
I did a node rotation and the node list was broken during after a new node was created.

**Review by sniok — APPROVED** [review]
Thanks! Really appreciate updated test cases and type definitions

File: frontend/src/components/cluster/Charts.tsx (hunk 1 above)
Comment (farodin91) [reply frontend/src/components/cluster/Charts.tsx:22]: I will change this.

**Comment by farodin91** [comment]
@sniok updated.

**Pull request merged**